mailbox/api/src/test/java/org/apache/james/mailbox/events/EventDeadLettersContract.java

```diff
@@ -460,7 +460,17 @@ default void groupsWithFailedEventsShouldReturnEmptyWhenNoStored() {
         }
     }
 
-    interface EventDeadLettersHealthCheckContract extends EventDeadLettersContract {
+    interface HasEventContract extends EventDeadLettersContract {
```

Comment:
Don't forget to change the name of this contract and tests when you change `hasEvent` method to `containsEvents` as @chibenwa suggested you :)